package.json

```diff
@@ -19,38 +20,43 @@
   "license": "MPL2",
   "dependencies": {
     "alameda": "^1.0.0",
+    "fxos-mvc": "fxos-eng/mvc#master",
```

Comment:
Moved all dev dependencies to the appropriate section. iirc dependencies section should contain only run time app dependencies.